stdlib/float.mli

```diff
@@ -60,7 +60,7 @@ external div : float -> float -> float = "%divfloat"
 external fma : float -> float -> float -> float =
   "caml_fma_float" "caml_fma" [@@unboxed] [@@noalloc]
 (** A fused multiply-add [fma x y z] computes [x * y + z] with a
-   single rounding. Deprecated if no hardware FMA support is provided,
+   single rounding. If no hardware support is provided,
```

Comment:
Ok, so IIUC, the criterion is not so much that the hardware support is provided or not, but whether the libc provides a compliant version (emulated or hardware based) or not.  There can be cases where hardware support is available but emulation is still used (your emulation, or from the libc).  (Sorry to nitpick, but it's important to explain as precisely possible sources of differences in fp calculations across different machines.)

Reply:
Yes, you're absolutely right! I'll fix it in the next commit.